### PR TITLE
Fix excluding more than two labels on issues list

### LINKED ISSUE #14840: bug: Excluding more than two labels does not work
- Gitea version (or commit ref): 1.13.2
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL) https://try.gitea.io/MGChecker/DokuForge2/pulls?q=&type=all&sort=&state=open&labels=-12523%2c-12525&milestone=0&assignee=0
  - [ ] No

## Description

If you try to exclude labels by using Alt+Click, after doing this with the third one the exclusion of the first one you selected is removed.

### PR DESCRIPTION
**Pull request by zeripath**
Fix #14840

Signed-off-by: Andrew Thornton <art27@cantab.net>

### CODE REVIEW

**Comment by noerw** [comment]
blocked by https://github.com/zeripath/gitea/pull/9, otherwise LGTM

**Review by noerw — APPROVED** [review]

File: web_src/js/index.js [hunk 1]
```diff
@@ -3753,36 +3753,30 @@ function initIssueList() {
       fullTextSearch: true
     });
 
+  function removeLabel (item) {
```
Comment (jaqra) [inline web_src/js/index.js]:
```suggestion
  function excludeLabel (item) {
```

**Review by ghost — APPROVED** [review]
Small nit. otherwise lgtm. just remove can be confusing

File: web_src/js/index.js [hunk 2]
```diff
@@ -3753,36 +3753,30 @@ function initIssueList() {
...
-        const item = $(selectedItems[0]);
-
-        const href = item.attr('href');
-        const id = item.data('label-id');
-
-        const regStr = `labels=(-?[0-9]+%2c)*(${id})(%2c-?[0-9]+)*&`;
-        const newStr = 'labels=$1-$2$3&';
-
-        window.location = href.replace(new RegExp(regStr), newStr);
+        removeLabel($(selectedItems[0]));
```
Comment (zeripath) [inline web_src/js/index.js]:
```suggestion
        excludeLabel($(selectedItems[0]));
```

**Comment by jaqra** [comment]
@zeripath sorry for bothering suggestion

**Comment by zeripath** [comment]
> @zeripath sorry for bothering suggestion

No don't worry - better to get this stuff fixed now.

**Pull request merged**